app/src/main/res/layout/pre_onboarding_comparison_chart.xml

```diff
@@ -129,7 +129,7 @@
         app:layout_constraintEnd_toStartOf="@id/feature2"
         app:layout_constraintStart_toStartOf="parent"
         app:layout_constraintTop_toTopOf="@id/feature2"
-        app:srcCompat="@drawable/ic_shield_check_color_24" />
+        app:srcCompat="@drawable/ic_shield_color_24" />
```

Comment:
Do we want the new shield in the onboarding comparison chart in production? Just double-checking.

Reply:
Yes, new new new icon for everyone.